internal/cli/cmd/revoke/revoke.go

```diff
@@ -67,6 +69,11 @@ kargo revoke --project=my-project --role=my-role --resource-type=stage --verb=up
 # Revoke permission to promote to stage dev from my-role
 kargo revoke --project=my-project --role=my-role \
   --resource-type=stage --resource-name=dev --verb=promote
+
+# Revoke my-role from users with a non standard claim and in a specific group
+# Please take note that even though you can specify subs, emails and groups in the generic claim
+# flag, if both are set, the bespoke sub, group, and email flags take precedence
+kargo revoke --project=my-project --role=my-role --claim=group=admins --claim=given_name=alice
```

Comment:
Honestly, I think I'd drop the existing, dedicated --email, --sub, and --group flags. More options sometimes == more confusion, and in this case, the documentation for the command actually becomes a whole lot more concise by dropping those redundant bits.

Same applies for related commands like grant.

Reply:
Yes makes sense, especially when I had to specify in the examples which takes precedence.

Reply:
Removed the email, group and sub flags.